server/@types/express/index.d.ts

```diff
@@ -22,6 +24,7 @@ export declare global {
       verified?: boolean
       id: string
       logout(done: (err: unknown) => void): void
+      systemClientToken: string
```

Comment:
I think `activeCaseloadId` should go into the User object, populated by populateCurrentUser middleware, but I've created a ticket to do this after this PR is in.